modules/codemod/lib/v6/index.ts

```diff
@@ -21,5 +21,7 @@ export default function transformer(file: FileInfo, api: API, options: Options)
     renameCanvasDepthValue,
   ];
 
-  return fixes.reduce((source, fix) => fix({...file, source}, api, options), file.source);
-}
+  return fixes.reduce((source, fix) => fix({...file, source}, api, options) as string, file.source);
```

Comment:
`as string` is needed because the `Transform` type enforces the return type which `reduce` is fumbling over because initial value is a string.